Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -1394,8 +1394,12 @@ def add_filename_suffix(file_names: list, file_extension: str):
     return ret_value
 
 
+
 def rasterize_command():  # pragma: no cover
     urls = demisto.getArg("url")
+    #in case of list from the war room - convert python-like list to list
+    if isinstance(urls, str) and urls.strip().startswith('['):
+        urls  = argToList(urls)
```

Comment:
I think what should be done here is to send the URLs to argToList anytime, if there is only one it will return a list of 1 else it will return all items in the list.